app/controllers/hosts_controller.rb

```diff
@@ -60,6 +60,9 @@ def index(title = nil)
       end
       format.yaml { render :text => search.all(:select => "hosts.name").map(&:name).to_yaml }
       format.json
+      format.csv do
+        send_data search.to_csv(columns: [:id, :name, :os, :environment, :model, :hostgroup, :last_report])
```

Comment:
can we add all attributes by default? I think most users won't mind to remove data vs adding would be more complex?